File: crates/aft/tests/fixtures/inspect_complexity/shaped.py
def shaped(flag: bool, n: int) -> int:
    if flag and n > 0:
        for value in range(n):
            _ = value

    match n:
        case 0:
            return 0
        case 1:
            return 1
        case _:
            return n if flag else -n

File: crates/aft/tests/fixtures/inspect_complexity/shaped.rs
pub fn shaped(flag: bool, n: i32) -> i32 {
    if flag && n > 0 {
        for value in 0..n {
            let _ = value;
        }
    }

    match n {
        0 => 0,
        1 => 1,
        _ => 2,
    }
}
